pkg/bloomgateway/bloomgateway.go

```diff
@@ -419,16 +393,18 @@ outer:
 // task is closed by the worker.
 // Once the tasks is closed, it will send the task with the results from the
 // block querier to the supplied task channel.
-func consumeTask(ctx context.Context, task Task, tasksCh chan<- Task, logger log.Logger) {
-	logger = log.With(logger, "task", task.ID)
+func (g *Gateway) consumeTask(ctx context.Context, task Task, tasksCh chan<- Task) {
+	logger := log.With(g.logger, "task", task.ID)
 
 	for res := range task.resCh {
 		select {
 		case <-ctx.Done():
 			level.Debug(logger).Log("msg", "drop partial result", "fp_int", uint64(res.Fp), "fp_hex", res.Fp, "chunks_to_remove", res.Removals.Len())
+			g.metrics.chunkRemovals.WithLabelValues("dropped").Add(float64(res.Removals.Len()))
 		default:
 			level.Debug(logger).Log("msg", "accept partial result", "fp_int", uint64(res.Fp), "fp_hex", res.Fp, "chunks_to_remove", res.Removals.Len())
 			task.responses = append(task.responses, res)
+			g.metrics.chunkRemovals.WithLabelValues("accepted").Add(float64(res.Removals.Len()))
```

Comment:
I don't think the meaning of _dropped_ or _accepted_ is intuitive for the operators. Maybe we can add what they mean in the metric _help_? Or chose different label values?

Reply:
Does this description make sense to you? https://github.com/grafana/loki/pull/11983/commits/0431acd2419d1752529913b9db0f67c2ba46711c

Reply:
Yes, that makes perfect sense now? :) Thank you!